apps/web/app/services/client/api/timer/timer-log.ts

```
@@ -13,3 +13,45 @@ export async function getTimerLogs(
+	tenantId: string,
+	startDate: string | Date,
+	endDate: string | Date,
+	timeZone?: string
+}) {
+
+	if (!organizationId || !tenantId || !startDate || !endDate) {
+		throw new Error('Required parameters missing: organizationId, tenantId, startDate, and endDate are required');
+	}
+	const start = typeof startDate === 'string' ? new Date(startDate).toISOString() : startDate.toISOString();
+	const end = typeof endDate === 'string' ? new Date(endDate).toISOString() : endDate.toISOString();
+	if (isNaN(new Date(start).getTime()) || isNaN(new Date(end).getTime())) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)